### PR TITLE
Update Github action with matrix php 8.1 & 8.2

### PR DESCRIPTION
**Pull request by pascalheidmann**

### CODE REVIEW

**Review by AnnaDamm — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -13,13 +13,12 @@ jobs:
             matrix:
                 php:
                     - '8.1'
+                    - '8.2'
                 symfony-versions: [false]
                 include:
                     -   description: 'Symfony 5.4'
-                        php: '8.1'
                         symfony-versions: 5.4.*
                     -   description: 'Symfony 6.0'
```
Comment (alexislefebvre) [inline .github/workflows/ci.yaml:21]:
```suggestion
                    -   description: 'Symfony 6.3'
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -13,13 +13,12 @@ jobs:
...
-                        php: '8.1'
                         symfony-versions: 6.0.*
```
Comment (alexislefebvre) [inline .github/workflows/ci.yaml:22]:
```suggestion
                        symfony-versions: 6.3.*
```

**Review by AnnaDamm — APPROVED** [review]

**Review by alexislefebvre — APPROVED** [review]

**Comment by pascalheidmann** [comment]
@theofidry can you have a look? :)

**Comment by theofidry** [comment]
Thank you @pascalheidmann, sorry for the long delay.

**Pull request merged**